doc/index.rst

```diff
@@ -23,7 +23,7 @@ command line:
 
 .. code-block:: bash
 
-    composer require twig/extensions ~1.1.0
+    composer require twig/extensions ~1.2.0
```

Comment:
I suggest changing it to ``~1.2`` instead, to benefit from semver properly (or even omitting the version entirely and letting composer guess it)